### PR TITLE
Better skinsuit

### PR DESCRIPTION
**Pull request by xboxnoob**
## About The Pull Request

It makes the skinsuits slow you down less and slightly more viable compared to hardsuits and firesuits
## Changelog
:cl:
tweak: tweaked skinsuit speed so that its not absolutely god awful but yet not so good that someone use it over the firesuit
/:cl:

### CODE REVIEW

**Comment by xboxnoob** [comment]
sorry about the extra commits i had to revert the map left over from my previous PR

**Comment by ike709** [comment]
It's intentionally godawful.

**Comment by xboxnoob** [comment]
not my idea zesk was being a lazy fuck and told me to do it

**Comment by zeskorion** [comment]
> not my idea zesk was being a lazy fuck and told me to do it



> It's intentionally godawful.

it's godawful to the point where non- ipc/plasmamen are complaining about it being useless, as the cold kills them faster than if they just ran through suitless

**Comment by zeskorion** [comment]
> not my idea zesk was being a lazy fuck and told me to do it

ike man says 2.5 and it's good

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -434,7 +434,7 @@ Contains:
 	icon = 'icons/obj/clothing/suits.dmi'
 	alternate_worn_icon = 'icons/mob/suit.dmi'
 	icon_state = "skinsuit_rolled"
-	slowdown = 3 //Higher is slower
+	slowdown = 2 //Higher is slower
```
Comment (Crossedfall) [inline code/modules/clothing/spacesuits/miscellaneous.dm:437]:
```suggestion
	slowdown = 2.5 //Higher is slower

```

**Pull request closed without merging**